app/components/results-page.js

```diff
@@ -1,4 +1,33 @@
 import Component from "@ember/component";
+import { computed } from "@ember/object";
 
 export default Component.extend({
+  score: null,
+  stepsForward: computed("answers", function() {
+    const numSteps = this.answers.filter(a => a === 1).length;
+    if (numSteps === 1) {
+      return `${numSteps} step`;
+    } else {
+      return `${numSteps} steps`;
+    }
+  }),
+  stepsBack: computed("answers", function() {
+    const numSteps = this.answers.filter(a => a === -1).length;
+    if (numSteps === 1) {
+      return `${numSteps} step`;
+    } else {
+      return `${numSteps} steps`;
+    }
+  }),
+  percentage: computed("answers", function() {
+    const allScores = this.model.content.map(i => i._record.score);
```

Comment:
This was the only way I could figure out how to access the actual records from the DB. It is soooo ugly but it works! I feel like there must be a better way to do this. 😳